test/core/event_engine/test_suite/tests/server_test.cc

```diff
@@ -174,7 +175,8 @@ TEST_F(EventEngineServerTest,
   args = args.Set(GRPC_ARG_RESOURCE_QUOTA, quota);
   ChannelArgsEndpointConfig config(args);
   auto listener = *test_ee->CreateListener(
-      std::move(accept_cb), [](absl::Status /*status*/) {}, config,
+      std::move(accept_cb),
+      [](absl::Status status) { ASSERT_TRUE(status.ok()); }, config,
```

Comment:
in the event that these assertions fail, we probably want to discover the status?

Reply:
Good idea, done.